src/Views/Filter/Subject.cshtml

```diff
@@ -30,10 +30,10 @@
             <h1 class="govuk-heading-xl">Find courses by subject</h1>
           </legend>
           <p class="govuk-body">Select the subjects you want to teach.</p>
-          <p class="govuk-body">You can train to teach a subject even if you don’t have a degree in it, or if you haven’t used your degree knowledge recently. For some subjects you can take a subject knowledge enhancement (SKE) course.</p>
-          <p class="govuk-body"><a href="https://getintoteaching.education.gov.uk/explore-my-options/teacher-training-routes/subject-knowledge-enhancement-ske-courses">Find out more about SKE courses on Get into Teaching</a></p>
-          <h2 class="govuk-heading-m">Your choice of subject will affect the financial support</h2>
-          <p class="govuk-body">For some subjects you could be eligible for up to £28,000 tax-free while you train, through a bursary or scholarship. Alternatively, salaried courses are available in many subjects (salaries start at £16,000). Student finance is available for all subjects.</p>
+          <h2 class="govuk-heading-m">Get more financial support</h2>
+          <p class="govuk-body">Most courses charge a fee. The financial support options available to you will depend on the subject you choose.</p>
+          <p class="govuk-body">For some subjects you could be eligible for up to £28,000 tax-free while you train, through a bursary or scholarship.</p>
```

Comment:
suggestion - can we generate the amount `£28,000` dynamically, from a `Math.Max(...)` on the subject list? we already have the data in memory and this would guarantee consistentcy

Reply:
Don't think it worth the overhead at this late stage.